### ISSUE TITLE
--log-output=none prevents debugging errors to show up

### ISSUE THREAD

**Issue by rgordill** [grafana/k6-operator#159]
### Brief summary

This option is automatically populated by k6-operator on the created jobs. This lead to several situations where the only way to debug what is happening is creating manually the same jobs without this option.

See for example https://github.com/grafana/k6-operator/issues/137#issuecomment-1223012236. In my case, it was another issue related with permissions, which I only find out after disabling the option. I have also tried using K6_LOG_OUTPUT in env vars without success.

### k6-operator version or image

latest (8.0.0rc3)

### K6 YAML

```yaml
apiVersion: k6.io/v1alpha1
kind: K6
metadata:
  name: k6-sample-with-extensions
  namespace: k6-tests
spec:
  parallelism: 4
  script:
    configMap:
      name: sample-stress-test
      file: test.js
  arguments: --http-debug -o output-prometheus-remote
  runner:
    image: xxx:latest
    env:
      - name: K6_LOG_OUTPUT
        value: stdout
```

### Steps to reproduce the problem

Launch the tests with any issues

### Expected behaviour

Log explaining the issue

### Actual behaviour

Nothing

**Comment by yorugac** [comment]
Hi @rgordill, thank you for the issue!

This is a tricky one because there is no good way to resolve it at the moment :disappointed: `--log-output=none` was added to allow tests with warnings to proceed in k6-operator but it does make troubleshooting problematic as you described. I believe this is best solved at k6 level, with addition of ability to turn off warnings as an output of at least `k6 archive` and `k6 inspect`, which k6-operator depends on. Most similar k6 issue on topic is https://github.com/grafana/k6/issues/2681 
The alternative is to add a hacky parsing of output to initializer in k6-operator which I'd prefer to avoid.

BTW, `K6_LOG_OUTPUT` does not overwrite CLI due to [k6 order of precedence](https://k6.io/docs/using-k6/k6-options/how-to/#order-of-precedence) -- that's expected behaivor.

**Comment by yorugac** [comment]
I believe this was fixed in #174 as well.
